### PR TITLE
use std:: for some math functions

### PR DESCRIPTION
**Pull request by zingale**
## PR checklist

- [ ] test suite needs to be run on this PR
- [ ] this PR will change answers in the test suite to more than roundoff level
- [ ] all newly-added functions have docstrings as per the coding conventions
- [ ] the `CHANGES` file has been updated, if appropriate
- [ ] if appropriate, this change is described in the docs

### CODE REVIEW

**Review by maxpkatz — APPROVED** [review]

File: Diagnostics/DustCollapse/main.cpp [hunk 1]
```diff
@@ -105,15 +105,15 @@ int main(int argc, char* argv[])
         auto x_maxdist = fmax(fabs(problo[0]), fabs(probhi[0]));
         auto y_maxdist = fmax(fabs(problo[1] - yctr), fabs(probhi[1] - yctr));
```
Comment (yut23) [inline Diagnostics/DustCollapse/main.cpp:106]: Should these be updated too?

Comment (zingale) [reply Diagnostics/DustCollapse/main.cpp:106]: yeah, probably.  I'll do a pass on those and add it to this PR

Comment (yut23) [reply Diagnostics/DustCollapse/main.cpp:106]: I also see a bunch of places we can use `powi`, but I figure we should do that in another PR

Comment (zingale) [reply Diagnostics/DustCollapse/main.cpp:106]: yes, that should be separate

**Pull request merged**